crates/esthri-cli/Cargo.toml

```diff
@@ -51,8 +51,6 @@ tokio = { version = "1.23.0", features = ["rt-multi-thread", "signal", "sync"] }
 tokio-util = { version = "0.7.8", features = ["compat", "codec"] }
 warp = "0.3"
 
-aws-sdk-s3 = { workspace = true }
-
```

Comment:
Since we pull in esthri, we can remove `aws-sdk-s3` dependency.
I'm thinking since esthri is tightly coupled with aws-sdk-s3 basically building on top of that, we should maybe export that to tightly couple the version as well for people using this library